staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5867,8 +5867,6 @@ const (
 type NodeConditionType string
 
 // These are valid but not exhaustive conditions of node. A cloud provider may set a condition not listed here.
-// The built-in set of conditions are:
-// NodeReachable, NodeLive, NodeReady, NodeSchedulable, NodeRunnable.
```

Comment:
NodeReady and NodeSchedulable still exist and are in use

Reply:
@AkihiroSuda please address @liggitt's comment, thanks.

Reply:
`NodeSchedulable` is not defined as a `NodeConditionType`.
It is defined as a string in `pkg/kubelet/events` though.

Reply:
(And it is obvious that `NodeReady` is defined as a built-in condition here)

Reply:
Worded as `Relevant events contain "NodeReady", "NodeNotReady", "NodeSchedulable", "NodeNotSchedulable".`.

Also updated `pkgs/apis/core/types.go`.